model/dex.go

```diff
@@ -0,0 +1,43 @@
+package model
+
+import (
+	"net/url"
+)
+
+type Dex struct {
+	Enabled         bool             `yaml:"enabled"`
+	Url             string           `yaml:"url"`
+	ClientId        string           `yaml:"clientId"`
+	Username        string           `yaml:"username"`
+	Groups          string           `yaml:"groups,omitempty"`
+	CaFile          string           `yaml:"caFile,omitempty"`
```

Comment:
If we use signed certificates and leaving CaFile empty, will not create the flag for ca. Maybe using a bool instead will be better.